content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -101,6 +101,28 @@ export class DiscoveryStreamAdmin extends React.PureComponent {
     return isMatch;
   }
 
+  renderFeedData(url) {
```

Comment:
We are now showing feed data, along with spoc data, to test that the score changes.